src/main/java/gregtech/api/unification/crafttweaker/CTMaterialHelpers.java

```diff
@@ -15,13 +17,16 @@
         return components == null || components.length == 0 ? ImmutableList.of() : ImmutableList.copyOf(components);
     }
 
-    protected static Material.FluidType validateFluidType(String fluidTypeName) {
-        if (fluidTypeName == null || fluidTypeName.equals("fluid")) return Material.FluidType.FLUID;
-        else if (fluidTypeName.equals("gas")) return Material.FluidType.GAS;
-        else {
-            CraftTweakerAPI.logError("Fluid Type must be either \"fluid\" or \"gas\"!");
+    protected static FluidType validateFluidType(String fluidTypeName) {
+        if (fluidTypeName == null || fluidTypeName.equals("fluid"))
+            return FluidTypes.LIQUID;
+
+        FluidType type = FluidType.getByName(fluidTypeName);
+        if (type == null) {
+            CraftTweakerAPI.logError("Fluid Type must be either \"liquid\" or \"gas\" or \"acid\"!");
```

Comment:
What about Plasma?

Reply:
Plasma is not intended to be specified in the FluidProperty, it is instead determined by the PlasmaProperty and should not be used here.

Reply:
This is not the FluidProperty though. This method checks that the FluidType is valid, and plasma is a valid FluidType that could be passed into this method